iot-hub/Quickstarts/InvokeDeviceMethod/Program.cs

```diff
@@ -19,15 +20,40 @@ internal class Program
         private static ServiceClient s_serviceClient;
         
         // Connection string for your IoT Hub
-        // az iot hub show-connection-string --hub-name {your iot hub name} --policy-name service
+        // az iot hub connection-string show --policy-name service --hub-name {YourIoTHubName}
+        // Override by passing service connection string as first parameter in command line
         private static string s_connectionString = "{Your service connection string here}";
 
+        // Default device name is `MyDotnetDevice`
+        // Override by passing custom name as second parameter in command line
+        private static string s_deviceName = "MyDotnetDevice";
```

Comment:
Instead, you can give the parameter a default value (in the attribute, which will make that value visible when someone runs --? when running the app.